### ISSUE TITLE
[Feature Request] Pull historical information to be able to display overall watch statistics

### ISSUE THREAD

**Issue by PiperMp3** [Boerderij/Varken#182]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I am using stock Varken + Grafana Dashboard that came with it.
Plex, Varken and Tautulli is on the same VM just FYI. I dont use any other module or add-on to Varken.

Using a TABLE visualizer, I would like a drop down which displays the different libraries reported by Tautulli.
Selecting one then populates the TABLE VIEW with the RatingKey (Unique ID of each file) + Title of the file (Some Episode/Some Movie/Some song) + more data.

**Describe alternatives you've considered**
I tried working with what was already in Varken + Tautulli, and I can easily aggregate different views to find the grand total of items inside a Library, but I am not able to find the actual ID/TITLE of the items inside the library.

**Additional context**
Basically, go to Tautulli -> Libraries, pick any, "Media Info" tab. You will see a list of each item inside the library chosen.
It will show all info Tautulli has for the item, so basically this grid view, inside Grafana.

I've tried "ratingKey", "Hash" "ID" "title" and other combos, without being able to find any hits.
Using tags for episodes, tracks, albums etc shows the number of unique items, but not the information for the items.
This leads me to believe Varken is not pulling this type of data from Tautulli.

I am more than happy to be proven wrong! :)

**Comment by samwiseg0** [comment]
So you would like to see all the items in a Library displayed as a table?

**Comment by PiperMp3** [comment]
Yes, just tried to explain it fully.
I use Tautulli as a data agent only, I don't really log into it anymore since I put up Varken. End goal is to have a table aggregate on most watched items over the x-period, times the item has been watched, etc. I just much prefer Elastic/Grafana world than clicking around on Tautulli.

**Comment by samwiseg0** [comment]
I believe there is a way you can do that with the data that is collected from the current activity. You can modify that table to show top items in the database but that would not be as accurate. 

Unfortunately, what you are asking for is not as simple as adding a ratingkey. We would have to pull all the history from Tautulli in the last X days or use just what is on the main page watch statistics. Currently there are no plans in implement either of those things as Varken is not intended to show historical information but just a dashboard of what is current/active. We do not want to recreate the same views/interface as Tautulli itself.

@DirtyCajunRice Do you have any thoughts?

**Comment by DirtyCajunRice** [comment]
Agree with sam. If you want to use tau as a db this is not the route you will find success in.
